docs/troubleshooting-guide/ldap.md

```diff
@@ -0,0 +1,56 @@
+### LDAP sample configuration
+
+In case you are running an LDAP server with an admin user of `admin` and a password like `adminpassword`, a root node of LDAP tree like `dc=example,dc=org` (example.org), user organizational unit like `users` and user group like `readers`, the configuration you need to include in Metabase is the following:
+
+USERNAME OR DN: `cn=admin,dc=example,dc=org`
+PASSWORD: `adminpassword`
+USER SEARCH BASE: `ou=users,dc=example,dc=org`
+USER FILTER: `(&(objectClass=inetOrgPerson)(|(uid={login})))` // could be left with the default value which will look for the user id both in the uid or email field
+GROUP SEARCH BASE: `cn=readers`
+
+### Related software for troubleshooting
+
+In case you run into an issue, make sure that you can login and use your LDAP directory with a software like [Apache Directory Studio](https://directory.apache.org/studio/). With it you can see the whole LDAP tree and check in the logs of your LDAP application the queries that are being done.
+
+### Current limitations
+
+- Metabase will populate the user profile with the name and surname a user has on LDAP on the first login. In case the user changes the name on the directory, it won't be automatically updated on Metabase.
```

Comment:
```suggestion
Metabase will populate the user profile with the name and surname a user has on LDAP on the first login. In case the user changes the name on the directory, it won't be automatically updated on Metabase.
```